ground-server/src/components/dashboard/dashboard.tsx

```diff
@@ -29,21 +29,29 @@ export function Dashboard({
     );
   };
 
-  const children = useMemo(() => {
-    return widgets.map((widget) => (
-      <DashboardWidget key={widget.layout.i}>{widget.children}</DashboardWidget>
-    ));
-  }, [widgets]);
+  // TODO for performance improvement, find a way that widgets does not rely on layout (then we can memoize the children and avoid re-rendering)
```

Comment:
Added a ticket

Reply:
It is also not necessary at all since we have such few widgets, and have very good performance still, so it should be VERY low priority. I just figured I would add a comment so that if we ever encounter a performance issue we can go and fix it.